examples/gym_ex/proxy/agent.py

```diff
@@ -30,13 +30,15 @@
 from aea.identity.base import Identity
 from aea.mail.base import Envelope
 
-sys.modules["packages.fetchai.connections.gym"] = locate(
-    "packages.fetchai.connections.gym"
-)
 from packages.fetchai.connections.gym.connection import (  # noqa: E402  # pylint: disable=wrong-import-position
     GymConnection,
 )
 
+
+sys.modules["packages.fetchai.connections.gym"] = locate(
+    "packages.fetchai.connections.gym"
+)
+
```

Comment:
I think the order needs to be retained as it was previously for this to work...

Reply:
OK